src/Core/Addon/Theme/Theme.php

```diff
@@ -271,4 +271,18 @@ private function getPageSpecificJs($pageId)
 
         return $js;
     }
+
+    /**
+     * Defines if the theme uses the core.js scripts or it provides it's own implementation.
+     *
+     * @return bool
+     */
+    public function requiresCoreScripts()
+    {
+        if ($this->attributes->get('theme_settings.core_scripts') === false) {
```

Comment:
Maybe you can just do like that : 

`return $this->attributes->get('theme_settings.core_scripts');`

😉

Reply:
Yeees, we said that with Valentin, I just pushed it and "Lets see what they say" 😄😄

If you guys to want to a direct call, I will change it.

OPINIONS PLS - @kpodemski @zuk3975

Reply:
https://media.giphy.com/media/bXvwCQglnTGKs/giphy.gif

Reply:
idk much about themes though :smile: 

```suggestion
        return $this->attributes->get('theme_settings.core_scripts', true)
```